src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseException.cs

```diff
@@ -5,23 +5,47 @@
 
 namespace System.Text.RegularExpressions
 {
+    /// <summary>
+    /// An exception as a result of a parse error in a regular expression <see cref="RegularExpressions"/>, with
+    /// detailed information in the <see cref="Error"/> and <see cref="Offset"/> properties.
+    /// </summary>
     [Serializable]
-    internal sealed class RegexParseException : ArgumentException
+    public sealed class RegexParseException : ArgumentException
     {
-        private readonly RegexParseError _error; // tests access this via private reflection
-
         /// <summary>Gets the error that happened during parsing.</summary>
-        public RegexParseError Error => _error;
+        public RegexParseError Error { get; }
 
         /// <summary>Gets the offset in the supplied pattern.</summary>
```

Comment:
```suggestion
        /// <summary>Gets the zero-based character offset in the regular expression pattern where the parse error occurs.</summary>
```
or something like that.

Reply:
I think this is related to the previous comment?

Reply:
I don't understand your reply. I made a suggested change.

Reply:
@stephentoub, sorry, I'm in mobile. I'm OK with your suggested changes, but github doesn't give me all the tools here. Hope clicking the "resolve" button accepts it automatically?

Reply:
@stephentoub, clearly something went wrong from mobile. I've now made sure it's in.